client/interfaces.go

```diff
@@ -133,9 +134,10 @@ func (client *Client) Connect(plugSnapName, plugName, slotSnapName, slotName str
 }
 
 // Disconnect breaks the connection between a plug and a slot.
-func (client *Client) Disconnect(plugSnapName, plugName, slotSnapName, slotName string) (changeID string, err error) {
+func (client *Client) Disconnect(plugSnapName, plugName, slotSnapName, slotName string, forget bool) (changeID string, err error) {
```

Comment:
it might be better to follow the style of other methods and have a DisconnectOptions struct